### PR TITLE
Query the new chart data api for charts that support it

### PR DESCRIPTION
**Pull request by suddjian**
### CATEGORY

- [ ] Bug Fix
- [x] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY

Charts with the metadata value `useLegacyApi` set to `false` will now query the new api endpoint. This is demonstrated with the word cloud plugin, which has been upgraded to the non-legacy version.

### TEST PLAN

I've added a test for the api query, and verified both the new word cloud and a few of the legacy chart types in Explore. Still verifying that charts work correctly in Dashboards. You're encouraged to try out the branch.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/package.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
     "@superset-ui/legacy-plugin-chart-sunburst": "^0.13.5",
     "@superset-ui/legacy-plugin-chart-table": "^0.13.5",
     "@superset-ui/legacy-plugin-chart-treemap": "^0.13.3",
-    "@superset-ui/legacy-plugin-chart-world-map": "^0.13.3",
+    "@superset-ui/legacy-plugin-chart-world-map": "^0.13.6",
```
Comment (villebro) [inline superset-frontend/package.json:87]: We also need to bump `@superset-ui/plugin-chart-word-cloud` to `0.13.5`

Comment (rusackas) [reply superset-frontend/package.json:87]: I went ahead and did this :)

Comment (rusackas) [reply superset-frontend/package.json:87]: Hope that's not a faux pas

**Review by villebro — APPROVED** [review]
🎉

**Review by rusackas — APPROVED** [review]
LGTM!

File: superset-frontend/package.json (hunk 1 above)
Comment (villebro) [reply superset-frontend/package.json:87]: Oops, I meant so say `0.13.6`

**Pull request merged**